tests/src/Kernel/Paragraphs/ContentRowInpageNavigationTest.php

```diff
@@ -0,0 +1,93 @@
+        ]),
+        5 => Paragraph::create([
+          'type' => 'oe_rich_text',
+          'field_oe_title' => 'Title rich text example 6',
+          'field_oe_text_long' => 'Curabitur molestie tempor sem, nec posuere nisl interdum blandit. Duis vestibulum, sapien sed sodales tempor, ipsum libero consectetur dolor, eget sagittis odio lectus eu felis. Fusce suscipit augue eget lorem consectetur, a efficitur nisi ornare. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Vivamus consequat erat quis libero maximus semper. Pellentesque eget erat venenatis tellus pulvinar lacinia. Donec luctus justo dolor, quis tempor ante aliquet vel. Curabitur non luctus nibh. Maecenas commodo dolor ac bibendum fermentum. Duis id suscipit justo. Vivamus lobortis eu tellus in tincidunt. Integer tempus augue ac dui tincidunt, sed blandit lectus imperdiet. Aenean sodales elit sit amet urna lacinia, non commodo dolor porta. Donec ultrices quam eget convallis interdum. Vestibulum egestas malesuada dui sed rhoncus. Sed finibus egestas metus vitae volutpat.',
+        ]),
+      ];
+    // Create content row paragraph with inpage navigation variant.
+    $paragraph = Paragraph::create([
+      'type' => 'oe_content_row',
+      'field_oe_title' => 'Page content',
+      'field_oe_paragraphs' => $sub_paragraphs,
+      'oe_paragraphs_variant' => 'inpage_navigation',
+    ]);
+    $paragraph->save();
+
+    // Testing: Inpage navigation - All paragraph types.
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+
+    $this->assertCount(1, $crawler->filter('div.row'));
+    $this->assertCount(1, $crawler->filter('div.col-md-3.d-none.d-md-block'));
+    $this->assertCount(1, $crawler->filter('nav.position-sticky'));
+    $this->assertCount(1, $crawler->filter('h5'));
+    $this->assertCount(1, $crawler->filter('ul.nav.nav-pills.flex-column'));
+    $this->assertCount(6, $crawler->filter('li.nav-item'));
+    $this->assertCount(6, $crawler->filter('a.nav-link'));
+    $this->assertCount(1, $crawler->filter('div.col-md-9'));
+    $this->assertCount(6, $crawler->filter('p'));
+
+    for ($i = 1; $i <= 6; $i++) {
```

Comment:
for loops in test have the tendency of masking failing scenarios, not necessarily the case here but I would not make a habit out it, let's have less items to test and list them one by one instead of a loop